infra/metadata.yaml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: blueprints.cloud.google.com/v1alpha1
+kind: BlueprintMetadata
```

Comment:
**Question:** Neat. Is this `BlueprintMetadata` CRD documented somewhere (I couldn't find anything on Moma)? This is my first time seeing a `blueprints.cloud.google.com/v1alpha1` resource.

Reply:
Documented [here](https://pkg.go.dev/github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli@v1.1.5/bpmetadata#BlueprintMetadata).

Reply:
Awesome, thanks, Awais!